uint8 state
uint64 DroneID
uint8 Run = 1
uint8 Stop = 2
uint8 Land = 3
uint8 Wait = 4
